tbx/settings/base.py

```diff
@@ -580,3 +580,24 @@
     "birdbath.processors.users.UserPasswordAnonymiser",
 ]
 WILLOW_OPTIMIZERS = True
+
+
+# Rich text settings
+WAGTAILADMIN_RICH_TEXT_EDITORS = {
+    "default": {
+        "WIDGET": "wagtail.admin.rich_text.DraftailRichTextArea",
+        "OPTIONS": {
+            "features": [
+                "h2",
+                "h3",
+                "h4",
+                "bold",
+                "italic",
+                "ul",
+                "ol",
+                "link",
+                "document-link",
```

Comment:
**Question (non-blocking)**: Do we want to add `superscript`?

Reply:
Let's leave it for now unless we get asked for it, as I think it may involve further styling updates - unless you know it was in the old site?